modules/csvparser/csvparser.c

```diff
@@ -145,13 +149,6 @@ csv_parser_process(LogParser *s, LogMessage **pmsg, const LogPathOptions *path_o
   gboolean result = csv_scanner_is_scan_finished(&scanner);
   csv_scanner_deinit(&scanner);
 
-  if (!result)
-    {
-      msg_debug("csv-parser() failed to parse all columns from the message, and drop-invalid flag is specified",
```

Comment:
The fact that the parsing was unsuccessful is still interesting. Shouldn't we emit a log here?

Reply:
The unsuccessful parsing will handle from general other place - here:
https://github.com/balabit/syslog-ng/blob/06a54a8020328ae7308bb4fa4359df25d24e90fa/lib/parser/parser-expr.c#L95